commcare_connect/opportunity/export.py

```diff
@@ -98,12 +99,33 @@ def _schema_sort(item):
 
 
 def export_empty_payment_table(opportunity: Opportunity) -> Dataset:
-    headers = ["Username", "Phone Number", "Name", "Payment Amount"]
+    headers = [
+        "Username",
+        "Phone Number",
+        "Name",
+        "Payment Accrued",
+        "Payment Completed",
+        "Payment Amount",
+        "Paid date",
```

Comment:
nit: this should be "Date of Payment" or "Payment Date" to be consistent with the other headers

Reply:
Done